Web/wwwroot/styles.css

```
@@ -0,0 +1,999 @@
+  max-width: 380px;
+  text-align: center;
+  animation: fade-in 0.5s var(--ease);
+}
+
+.chat-msg {
+  max-width: 82%;
+  padding: 10px 14px;
+  border-radius: 12px;
+  font-size: 13.5px;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)